src/boot/efi/boot.c

```diff
@@ -2013,6 +2074,80 @@ static VOID config_entry_add_linux(
         uefi_call_wrapper(linux_dir->Close, 1, linux_dir);
 }
 
+static EFI_STATUS secure_boot_enroll(Config *config, ConfigEntry *entry) {
+        EFI_STATUS err = EFI_SUCCESS;
+        static const UINT32 sb_vars_opts =
```

Comment:
I think @poettering would rather see a #define for this... I certainly prefer it like this, though it does not need a static modifier.

Reply:
i'd drop both "static" and "const" here.